packages/core/src/external/aws/appConfig.ts

```diff
@@ -0,0 +1,30 @@
+import { AppConfig } from "aws-sdk";
+
+function makeAppConfigClient(region: string): AWS.AppConfig {
+  return new AppConfig({ region });
+}
+
+export async function getFeatureFlagValue<T>(
+  region: string,
+  appId: string,
+  configId: string,
+  envName: string,
+  featureFlagName: string
+): Promise<T | undefined> {
+  const appConfig = makeAppConfigClient(region);
+  const config = await appConfig
+    .getConfiguration({
```

Comment:
Is this cached? Or will we make an network request for every check of the flag?

Reply:
@leite08 I think the way AWS AppConfig works is it pushed the config to the hosts after deployment

so, the only network request is to localhost to get the config

Reply:
Its defs NADB now even if it doesn't cache, but we need to be sure. I'll add an item here to check on the next few days. 👍